src/fetchers/wakatime-fetcher.js

```diff
@@ -4,27 +4,32 @@ import { MissingParamError } from "../common/utils.js";
 /**
  * WakaTime data fetcher.
  *
- * @param {{username: string, api_domain: string, range: string}} props Fetcher props.
+ * @param {{username: string, api_domain: string}} props Fetcher props.
  * @returns {Promise<WakaTimeData>} WakaTime data response.
  */
-const fetchWakatimeStats = async ({ username, api_domain, range }) => {
+const fetchWakatimeStats = async ({ username, api_domain }) => {
   if (!username) throw new MissingParamError(["username"]);
 
+  // Retrieve user range.
+  const { data: userdata } = await axios.get(
+    `https://${
+      api_domain ? api_domain.replace(/\/$/gi, "") : "wakatime.com"
+    }/api/v1/users/${username}/stats`,
+  );
+  const range = userdata.data.range;
+
+  // Loop through available ranges to get user data.
```

Comment:
Seems like it has been fixed from the upstream. Do we still need the following request here?

Reply:
Good catch, I think the endpoint was overlooked in https://github.com/anuraghazra/github-readme-stats/commit/a340900e8a1fac282d953fd4c7ac1ae9a459726e.

Reply:
`/stats` is fine, but `/stats/` is not 😄